modules/data-entry/src/main/frontend/src/dataHomepage/DeleteButton.jsx

```diff
@@ -45,7 +45,8 @@ function DeleteButton(props) {
   const [ deletionInProgress, setDeletionInProgress ] = useState(false);
 
   const buttonText = label || ("Delete " + (entryType?.toLowerCase() || '')).trim();
-  const defaultDialogAction = `Are you sure you want to delete ${entryType} ${entryName}?`;
+  const defaultDialogMessage = `Are you sure you want to delete the following ${entryType}:`;
+  const defaultDialogAction = entryName;
   const defaultErrorMessage = `${entryName || "The item"} could not be removed.`;
```

Comment:
Would this be a better error message?

```suggestion
  const defaultErrorMessage = `The ${entryType || "item"} could not be removed.`;
```